hack/make/.build-deb/rules

```diff
@@ -6,7 +6,7 @@ override_dh_gencontrol:
 	# if we're on Ubuntu, we need to Recommends: apparmor
 	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
 	# if we are building experimental we reccomend yubico-piv-tool
-	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
+	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
```

Comment:
yeah if you ` apt-file search libykcs11.so` you get nothing as well...

On Thu, Jan 7, 2016 at 3:12 PM, Jessica Frazelle <
jessica.frazelle@docker.com> wrote:

> It's on their apt repo... Or ours now but I emailed them about it it's
> super weird
>
>
> On Jan 7, 2016, 15:05 -0800, Tianon Gravi <notifications@github.com>,
> wrote:
>
> In hack/make/.build-deb/rules
> <https://github.com/docker/docker/pull/19168#discussion_r49139101>:
>
> > @@ -6,7 +6,7 @@ override_dh_gencontrol:
> >    # if we're on Ubuntu, we need to Recommends: apparmor
> >    echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
> >    # if we are building experimental we reccomend yubico-piv-tool
> > -  echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
> > +  echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
>
> I see https://packages.debian.org/sid/yubico-piv-tool, but
> https://packages.debian.org/sid/libykcs11-1 is 404 -- where does this
> other package come from? [image: :confused:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19168/files#r49139101>.
>
>